Makefile

```diff
@@ -1,7 +1,7 @@
 # Current Operator version
-VERSION ?= 0.5.0
+VERSION ?= 0.8.1
 # Default bundle image tag
-BUNDLE_IMG ?= controller-bundle:$(VERSION)
+BUNDLE_IMG ?= quay.io/evryfs/github-actions-runner-operator:$(VERSION)
```

Comment:
Wont't this conflict with the actual image for the operator, or at least be confusing - is there any common pattern to follow?

Reply:
Pushed a change. You can set environment variables to override at runtime